code/modules/assembly/heartmonitor.dm

```diff
@@ -15,4 +15,7 @@
 		if(istype(holder.loc, /mob/living))
 			var/mob/living/L = holder.loc
 			if(L.stat || L.stunned || L.weakened)
-				pulse()
```

Comment:
Fragments of when I was trying to figure out how the signaler magically gets the usr from the other assemblies.